libs/pmp-api/api-service/repository/core/src/lib/queries/handlers/list-repository-users.handler.ts

```diff
@@ -0,0 +1,44 @@
+import { IQueryHandler, QueryBus, QueryHandler } from '@nestjs/cqrs';
+import { ListRepositoryUsersQuery } from '../list-repository-users.query';
+import {
+  PrModel,
+  RepositoryUserStatisticsReadModel
+} from '@pimp-my-pr/pmp-api/api-service/repository/domain';
+import { RepositoryDataService } from '@pimp-my-pr/pmp-api/api-service/repository/data-access';
+import { GetUserPrsQuery } from '../get-user-prs.query';
+import { QueryBusService } from '../../services/query-bus.service';
+
+@QueryHandler(ListRepositoryUsersQuery)
+export class ListRepositoryUsersHandler
+  implements IQueryHandler<ListRepositoryUsersQuery, RepositoryUserStatisticsReadModel[]> {
+  constructor(
+    private queryBus: QueryBus,
+    private queryBusService: QueryBusService,
+    private repositoryRepository: RepositoryDataService
+  ) {}
+
+  async execute(query: ListRepositoryUsersQuery): Promise<RepositoryUserStatisticsReadModel[]> {
+    const result: Promise<RepositoryUserStatisticsReadModel>[] = [];
```

Comment:
![image](https://user-images.githubusercontent.com/30656326/71893628-7f39ee80-314c-11ea-952a-ac93de9f6db2.png)
And maybe we should create some helper for flatMapping